docs/sql/statements/alter-role.rst

```diff
@@ -74,3 +78,34 @@ are supported to alter an existing user account:
 
    Passwords and JWT properties can be changed only for existing database
    users, but not to roles.
+
+:session settings:
+
+  Any of the modifiable :ref:`session settings <conf-session>`. The value set
+  is used for the user when logins to the database, instead of the default
+  value, thus, there is no need to use ``SET`` statements to modify the setting
+  value on its user session.
+
+
+.. NOTE::
+
+    The session settings can only be set to a user and not on a role and
+    are therefore are not inherited to other users.
```

Comment:
ALTER USER  and ALTER ROLE are the same thing in CrateDB as well.
but if you execute `ALTER ROLE <roleName>`, where `roleName` is a Role (not a user, so it cannot login) then you cannot set session settings to it, since they are not inherited, it doesn't make sense, and it can be confusing to set settings that have no effect.